MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -95,14 +100,36 @@ public class MediaInfoService : BaseApiService
             _networkManager = networkManager;
             _mediaEncoder = mediaEncoder;
             _userManager = userManager;
-            _json = json;
             _authContext = authContext;
         }
 
         public object Get(GetBitrateTestBytes request)
         {
-            var bytes = new byte[request.Size];
-            return ResultFactory.GetResult(null, bytes, "application/octet-stream");
+            const int MaxSize = 10_000_000;
+
+            var size = request.Size;
+
+            if (size <= 0)
+            {
+                throw new ArgumentException($"The requested size can't be equal or smaller than 0.", nameof(request));
```

Comment:
I would rather we log the requested size (or at least put it in the exception) for easier log-driven debugging